utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -11,6 +11,9 @@ import android.widget.TextView
 import com.bumptech.glide.request.target.CustomTarget
 import com.bumptech.glide.request.transition.Transition
 import javax.inject.Inject
+import android.opengl.ETC1.getWidth
+
+
```

Comment:
Remove these extra lines and unused imports.

Reply:
done.